tutorials/encrypt-s3-data-rclone/index.mdx

```diff
@@ -230,15 +230,15 @@ Choose a number from below, or type in your own value
     \ "standard"
   2 / Very simple filename obfuscation.
     \ "obfuscate"
-  3 / Don't encrypt the file names.  Adds a ".bin" extension only.
+  3 / Do not encrypt the file names.  Adds a ".bin" extension only.
```

Comment:
Should this be "Add" ?

Reply:
This a copy of the output of the tool. I re-edit the message back to what it was